stage0/README.md

```diff
@@ -37,3 +64,113 @@ To use the binary, pass it to `qemu -bios`; for example:
 ```shell
 qemu-system-x86_64 -nodefaults -nographic -no-reboot -machine microvm -bios stage0.bin
 ```
+
+## Loading the kernel
+
+There are two options for loading the kernel: either get the VMM to load it into
+guest memory before the VM even starts, or let stage0 load the kernel from
+`fw_cfg`. Both approaches have their benefits and drawbacks.
+
+**Note**: the kernel has to be an ELF file! If you want to use Linux, use the
+`extract-vmlinux` script to create the uncompressed `vmlinux` image from the
+`bzImage` file.
+
+### Pre-loading the kernel
```

Comment:
you could add "Option 1" here the emphasize this is one of the two options you mention in the intro paragraph.